### PR TITLE
[#11878] Update SearchAccountRequests endpoint 

### PR DESCRIPTION
**Pull request by EuniceSim142**
Part of #11878.

**Outline of solution**
- Add `comments` and `status` as search-able fields for account requests.
- Update `solr.sh` script to generate solr schema with new 'comments' and 'status' fields.

### CODE REVIEW

**Review by domoberzin — COMMENTED** [review]
Code looks good to me, but should some tests be added to `AccountRequestSearchIT` for these new fields?

**Comment by EuniceSim142** [comment]
IT tests have been added, and updated search action to use id to query db for account request entities.

AccountRequestSearchIndexing action also updated to not include `java.util.UUID:` for id field (Don't think it's necessary and makes converting from solr document field's value to UUID object easier)
| Before: Without `toString()` | Now: with `toString()` |
| -- | -- |
| Screenshot 2024-03-28 at 5 21 37 PM | Screenshot 2024-03-28 at 5 21 43 PM |

**Review by jayasting98 — COMMENTED** [review]
It seems okay to me, the failing check just needs to be addressed.

File: src/main/java/teammates/storage/sqlapi/AccountRequestsDb.java [hunk 1]
```diff
@@ -155,7 +155,7 @@ public void deleteDocumentByAccountRequestId(UUID accountRequestId) {
             // Solr saves the id with the prefix "java.util.UUID:", so we need to add it here to
```
Comment (domoberzin) [inline src/main/java/teammates/storage/sqlapi/AccountRequestsDb.java:155]: I think this comment can be removed since the changes have been made below

**Review by domoberzin — COMMENTED** [review]
Other than the one comment below, all good to me

**Review by domoberzin — APPROVED** [review]
LGTM!

**Review by ziqing26 — APPROVED** [review]
@EuniceSim142 @domoberzin Thanks a lot for managing to fix the failing tests! LGTM 🙏

**Pull request merged**